### ISSUE TITLE
Demonstrate API

### ISSUE THREAD

**Issue by shacker** [wagtail/bakerydemo#83]
Not sure we want to tackle this for v1, but we had discussed before the sprint the possibility of demonstrating the Wagtail REST API. 

We could consume an endpoint of breads with a simple jquery demo, or be more modern and use React, though that might involve bringing in webpack (though I tried a React demo once that provided a runtime compilation rather than using webpack - could look that up). 

Whatever we do, I feel we should keep it simple.

**Comment by heymonkeyriot** [comment]
Yup, this feels like something to implement after we've got the demo site up and running, potentially as a separate project if we want to do a nice front-end to it. I know a couple of people flagged they'd be interested in working on a small React app for it.

**Comment by heymonkeyriot** [comment]
Since we're doing post-launch have added `some day` label to this issue.

**Comment by thibaudcolas** [comment]
Follow-up: #545
